modules/optionals/ecr/stack.py

```diff
@@ -0,0 +1,42 @@
+import cdk_nag
+from aws_cdk import Aspects, Duration, Stack
+from aws_cdk import aws_ecr as ecr
+from constructs import Construct
+
+IMAGE_MUTABILITY = {
+    "IMMUTABLE": ecr.TagMutability.IMMUTABLE,
+    "MUTABLE": ecr.TagMutability.MUTABLE,
+}
+
+
+class EcrStack(Stack):
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        repository_name,
```

Comment:
How come none of the input parameters are typed?  How did this pass validate.sh?